climpred/tests/test_effective_p_value.py

```diff
@@ -12,7 +12,7 @@ def test_eff_sample_size_smaller_than_n_hind_da_initialized_1d(
     of the data."""
     N = hind_da_initialized_1d.mean('member').count('init')
     eff_N = compute_hindcast(
-        hind_da_initialized_1d, reconstruction_da_1d, metric='eff_n', max_dof=True,
+        hind_da_initialized_1d, reconstruction_da_1d, metric='eff_n', common='max_dof',
```

Comment:
should be `alignment='maximize'` here. This passed since it just thought it was a metric kwarg.